conf/output/test-hebis-to-lobid-output-11.json

```diff
@@ -0,0 +1,113 @@
+{
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "id" : "http://lobid.org/resources/090363000#!",
+  "type" : [ "BibliographicResource", "Book" ],
+  "medium" : [ {
+    "label" : "Print",
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010"
+  } ],
+  "title" : "Mainz transparent",
+  "almaMmsId" : "090363000",
+  "isbn" : [ "3874395340", "9783874395342" ],
+  "oclcNumber" : [ "57680814" ],
+  "dnbId" : "958645337",
+  "otherTitleInformation" : [ "2000 years of a city ; [city guide]" ],
+  "publication" : [ {
+    "startDate" : "2000",
+    "type" : [ "PublicationEvent" ],
+    "location" : [ "Mainz" ],
+    "publishedBy" : [ "Schmidt" ]
+  } ],
+  "sameAs" : [ {
+    "id" : "http://worldcat.org/oclc/57680814",
+    "label" : "OCLC Ressource"
+  }, {
+    "id" : "https://d-nb.info/958645337",
+    "label" : "DNB-Ressource"
+  } ],
+  "language" : [ {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/eng",
+    "label" : "Englisch"
+  } ],
+  "extent" : "127 S. : zahlr. Ill., graph. Darst., Kt.",
+  "natureOfContent" : [ {
+    "label" : "Führer",
+    "id" : "https://d-nb.info/gnd/4155569-7"
+  } ],
+  "subject" : [ {
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "Gemeinsame Normdatei (GND)",
+      "id" : "https://d-nb.info/gnd/7749153-1"
+    },
+    "label" : "Mainz",
+    "id" : "https://d-nb.info/gnd/4037124-4",
+    "gndIdentifier" : "4037124-4"
+  }, {
+    "notation" : "61",
+    "type" : [ "Concept" ],
+    "source" : {
+      "label" : "Systematik der DNB (bis 2003)",
+      "id" : "https://bartoc.org/en/node/18497"
+    }
+  }, {
```

Comment:
This `subject` entry also is to sparse wrt the lobid-resources JSON schema, see https://github.com/hbz/rpb/pull/105#issuecomment-2689971143

Reply:
Same situation as above.